auslib/web/admin/swagger/api.yaml

```diff
@@ -0,0 +1,441 @@
+
+schemes:
+  - http
+security:
+  - basic: []
+securityDefinitions:
+  basic:
+    type: basic
+    description: HTTP basic access authentication
+
+tags:
+  - name: "Rules"
+    description: "Balrog Rules Description."
+    externalDocs:
+      description: "Find out more about Rules"
+      url: "http://mozilla-balrog.readthedocs.io/en/latest/database.html#rules"
+
+  - name: "Users"
+    description: "Data about admin Users"
+    externalDocs:
+      description: "Find out more about users"
+      url: "http://mozilla-balrog.readthedocs.io/en/latest/database.html#users"
+
+  - name: "CSRF Token"
+    description: "CSRF Token Description"
+    externalDocs:
+      description: "Find out more about CSRF-Token"
+      url: "http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#csrf-token"
+
+# basePath: "/api"
+# host: "localhost:8080"
+# Swagger-UI bug : fix TODO week 13
+
+paths:
+  /csrf_token:
+    get:
+      summary: Returns valid csrf_token in response header.
+      description: Returns an empty response with a valid CSRF token in the X-CSRF-Token header.
+      tags:
+        - "CSRF Token"
+      operationId: auslib.web.admin.views.mapper.csrf_get
+      consumes: []
+      produces:
+        - text/html
+      deprecated: false
+      responses:
+        '200':
+          description: csrf token generate
+          headers:
+            X-CSRF-Token:
+              description: token
+              type: string
+
+  /users:
+    get:
+      summary: Returns all of Users in Balrog's DB
+      description: Returns all of the Rules in Balrog’s database inside of a JSON Object.
+      tags:
+        - Users
+      operationId: auslib.web.admin.views.mapper.users_get
+      consumes: []
+      produces:
+        - application/json
+      deprecated: false
+      responses:
+        '200':
+          description: Get all Users
+          schema:
+            type: object
+            properties:
+              users:
+                type: array
+                items:
+                  type: string
+                  minItems: 0
+                example: ["balrogadmin"]
+          #examples:
+          #  application/json:
+          #    users: ["balrogadmin"]
+# NOTE: 'examples' object in Response objects is no longer supported.
+# However it hasn't been documented properly in swagger docs
+# To see example values, set the 'example' value in the Model
+  /rules:
+    get:
+      summary: Returns all of Rules in Balrog's DB
+      description: Returns all of the Rules in Balrog’s database inside of a JSON Object.
+      tags:
+        - Rules
+      operationId: auslib.web.admin.views.mapper.rules_get
+      parameters:
+        - name: product
+          in: query
+          description: Name of the product to filter the rules against.
+          type: string
+          x-nullable: true
+          required: false
+        - name: format
```

Comment:
This can be addressed in a follow-up. Maybe we can file a new bug and make it a good first bug.